dtde/spark/cli/cluster_delete.py

```diff
@@ -12,7 +12,14 @@ def setup_parser(parser: argparse.ArgumentParser):
 
 def execute(args: typing.NamedTuple):
     cluster_id = args.cluster_id
+
+    confirmation_cluster_id = input("Please confirm the id of the cluster you wish to delete: ")
```

Comment:
As part of this change, we should have a --force flag to override this option as it may get annoying for some users (like me :))

Reply:
yeah. i think this is actually better than y/n

Reply:
I think we should have both. If we remove the y/n then we should not need a force flag.

Reply:
@jiata Are you saying we have users confirm by typing the cluster id (instead of y/n), and also have a --force flag?